_docs/technical/2017-02-16-delegation.md

```diff
@@ -78,6 +78,19 @@ blockchain. Issuer can post [only one
 certificate](https://github.com/input-output-hk/cardano-sl/blob/763822c4fd906f36fa97b6b1f973d31d52342f3f/src/Pos/Delegation/Logic/VAR.hs#L401)
 per one epoch.
 
+Furthermore, heavyweight delegation certificate contains issuer's and delegate's public keys.
```

Comment:
It's the case for both lightweight and heavyweight
I would move this to a separate section, like "Transaction distribution trick"